mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(preTax) {
+  return preTax * 1.2;
```

Comment:
Hi Malkit, You doing great even you cover the extra one. in calculateSalesTax function return you multiply preTax with 1.2. I think it should be 0.2.

Reply:
I agree Michael then name of the function could be more meaningful.